src/main/resources/templates/fragments/register-intro.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+  <body>
+    <th:block th:fragment="register-intro">
+      <div class="registers-intro">
+        <div class="grid-row">
+          <div class="column-two-thirds">
+            <h1 class="heading-medium">Using the <th:block th:utext="${friendlyRegisterId}">register name</th:block> API</h1>
+            <p>We provide a secure API that is easy to integrate with so you can get the most up-to-date data from across government.</p>
+            <p>Create an API key to ensure that you always access the most recent version of all registers. The data is provided as JSON and CSV.</p>
+            <p><a th:href="@{'https://www.registers.service.gov.uk/registers/' + ${registerId}}" class="button">Get started</a></p>
```

Comment:
Do we want to link directly e.g. to `https://www.registers.service.gov.uk/create-api-key?register=${registerId}`?

Reply:
We want to link to the FE page for the register.